samples/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -110,6 +117,26 @@ static void disconnected(struct bt_conn *conn, u8_t reason)
 	}
 }
 
+extern bool bt_le_conn_params_valid(const struct bt_le_conn_param *param);
+
+static bool le_param_req(struct bt_conn *conn, struct bt_le_conn_param *param)
+{
+	LOG_INF("Connection parameter update request");
+
+	param->interval_min = 1;
+	param->interval_max = 1;
+
+	if (bt_le_conn_params_valid(param)) {
+		LOG_INF("Low latency operation");
+	} else {
+		LOG_INF("Normal operation");
+		param->interval_min = 6;
```

Comment:
Please add a comment this means 7.5 ms

Reply:
I would leave it as is - the value is used everywhere in the stack.